test/pytest/test_packages.py

```diff
@@ -105,48 +106,54 @@ def test_priority(self):
         assert parsed['guest'] == {'description': 'Guest'}
 
     def add_conditional_manifest(self, name, conditions):
-        (self.package_dir / name).mkdir()
-        (self.package_dir / name / 'manifest.json').write_text(f'{{"conditions": [ {conditions} ] }}')
+        self.make_package(name, conditions=conditions)
 
     def test_conditions(self):
-        self.add_conditional_manifest('empty', '')
+        self.make_package('empty', conditions=[])
 
         # path-exists only
-        self.add_conditional_manifest('exists-1-yes', '{"path-exists": "/usr"}')
```

Comment:
Oops.  I did remove `add_conditional_manifest()` and even eluded to it in the commit log:

>  This let's us drop our existing helper which was specific to testing conditionals since the new one is sufficient.

but the fragment that actually did that ended up in the wrong commit.  I'll rebase that.